src/nvim/api/keysets.lua

```diff
@@ -29,6 +29,9 @@ return {
     "script";
     "expr";
     "unique";
+    "callback";
+    "cb";
```

Comment:
These two look duplicate, we should settle for one. (normally we would use `on_xx` for callback args though I am not sure what xx would be..)

Reply:
`cb` is just alias for callback . like set_hl can have fg/bg as alias.

I'm not sure if `on_xx` is appropriate here .

Reply:
> like set_hl can have fg/bg as alias.

This is the special case. It doesn't mean we should go around and add lot's of aliases to freshly designed API:s. Let's pick one or the other. We are expected to create an "ergonomic" layer above this for init.lua anyway.

Reply:
ok then . will stick to callback only